src/applications/mhv/secure-messaging copy/routes.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import App from '../secure-messaging/containers/App';
+
+const routes = (
+  <Switch>
+    <Route path="/" key="App">
+      <App isPilot />
```

Comment:
This right here is the core idea. This could be any sort of value including an object full of booleans to control certain features.